harness/tests/src/main/kotlin/godot/tests/Invocation.kt

```diff
@@ -78,7 +78,7 @@ class Invocation : Node3D() {
 
 	@Export
 	@RegisterProperty
-	lateinit var lateinitString: String
+	var lateinitString: String? = "null"
```

Comment:
Shouldn't it be `= null` instead of `= "null"` to test nullability ?

Reply:
yeah was a mistake from testing